### PR TITLE
[KEYCLOAK-17998] Upgrade PostgreSQL JDBC Driver

### PR DESCRIPTION
**Pull request by abstractj**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -131,7 +131,7 @@
         <mysql.version>8.0.23</mysql.version>
         <osgi.version>4.2.0</osgi.version>
         <pax.web.version>7.1.0</pax.web.version>
-        <postgresql.version>42.2.8</postgresql.version>
+        <postgresql.version>42.2.13</postgresql.version>
```
Comment (vramik) [inline pom.xml:134]: EAP7.4 certifies with jdbc driver version 42.2.18, is there any specific reason why we're upgrading to 42.2.13? Could we upgrade to 42.2.18 instead?

**Review by vramik — COMMENTED** [review]
Thank you @abstractj for the PR. Before running tests on postgres I have one question.

File: pom.xml (hunk 1 above)
Comment (abstractj) [reply pom.xml]: Thanks @vramik, there's no specific reason, so I updated the PR to the version you suggested.

**Review by vramik — APPROVED** [review]
Thank you @abstractj. I've run base testsuite with postgres (42.2.18 driver) locally and there is only one failure not related with this change (https://issues.redhat.com/browse/KEYCLOAK-14509)

**Review by hmlnarik — APPROVED** [review]

**Review by pskopek — APPROVED** [review]
LGTM!

**Comment by hmlnarik** [comment]
Thanks for the PR!

**Pull request merged**